static/usage/v8/radio/helper-error/demo.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Input</title>
+    <link rel="stylesheet" href="../../common.css" />
+    <script src="../../common.js"></script>
+    <script
+      type="module"
+      src="https://cdn.jsdelivr.net/npm/@ionic/core@8.4.4-dev.11741289071.1e8e1c3f/dist/ionic/ionic.esm.js"
+    ></script>
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/@ionic/core@8.4.4-dev.11741289071.1e8e1c3f/css/ionic.bundle.css"
+    />
+  </head>
```

Comment:
```suggestion

    <style>
      ion-radio-group {
        display: inline-block;
        width: 200px;
      }
    </style>
  </head>
```

This makes it so the error text doesn't make the radios shift when it changes.